STYLE-GUIDE.md

```diff
@@ -0,0 +1,202 @@
+|  Axelar’s __General Message Passing (GMP)__ enables a developer building on one chain to call any function on any other connected chain. For GMP to work, chains A and B must be EVM or Cosmos with a deployed Axelar Gateway contract.   |  Axelar’s __General Message Passing (GMP)__ enables a developer building on one chain to call any function on any other connected chain. For __GMP__ to work, chains A and B must be EVM or Cosmos with a deployed Axelar Gateway contract.  |
+
+### Underlined font
+
+* Do not underline anything in the documentation.
+
+### Code font
+
+* Use code font for text input, strings, and any names found in code, such as properties and methods.
+
+|  **Recommended**  |  **Not recommended** |
+| --- | --- |
+|  If the prepaid gas is insufficient, you might see `NOT ENOUGH GAS` or other messages, such as `Insufficient gas for executing the transaction` and `intrinsic gas too low`.  |  If the prepaid gas is insufficient, you might see __NOT ENOUGH GAS__ or other messages, such as __Insufficient gas for executing the transaction__ and __intrinsic gas too low__.  |
+|  `sendToMany()` takes many parameters, including `_destChain` and `_destContractAddr`.  |  **sendToMany()** takes many parameters, including **_destChain** and **_destContractAddr**.  |
+|  Give your test wallet an alias, such as `My Sepolia test wallet`.  |  Give your test wallet an alias, such as “My Sepolia test wallet”.  |
+
+### Hyperlinks
+
+* Use descriptive hyperlink text rather than the word  “here.”
+
+|  **Recommended**  |  **Not recommended** |
+| --- | --- |
+|  For more information on General Message Passing, check out [Axelar’s YouTube channel](https://www.youtube.com/channel/UCf8GFg58fdp1iZwLAOV1Tgg).  |  For more information on General Message Passing, check out Axelar’s YouTube channel [here](https://www.youtube.com/channel/UCf8GFg58fdp1iZwLAOV1Tgg).  |
+
+### Lists
+
+* When writing a list, either use complete sentences for all elements or no elements. Either every element has a period at the end, or none of them do. Do not mix and match.
+
+**Recommended**
+
+Implement the following to send an interchain transaction with Axelar:
+
+* `AxelarExecutable` – The contract to handle a message on the destination chain once a transaction has been sent to the Axelar network
+* `IAxelarGateway` – The Axelar Gateway
+* `IAxelarGasService` – The Axelar Gas Service
+* `IERC20` – The ERC-20 token interface to access ERC-20-related functionality
+
+**Not recommended**
+
+Implement the following to send an interchain transaction with Axelar:
+
+* `AxelarExecutable` – This is a contract that handle a message on the destination chain once a transaction has been sent to the Axelar network.
+* `IAxelarGateway` – The Axelar Gateway
+* `IAxelarGasService` – The Axelar Gas Service
+* `IERC20` – This represents the ERC-20 token interface to access ERC-20-related functionality.
```

Comment:
Can you add a section about capitalization? I've heard we don't capitalize "Axelar network" and other standards from Galen.

Reply:
There is [another set of content guidelines for marketing and partner comms](https://www.notion.so/bright-ambert-2bd/Axelar-comms-content-rules-0c003b259d274e839f77f92034fc127b#e283efd1a0634d6ea1688f528aa93423) written by Galen, some of which contradict the guidelines set here. I'll meet with him to discuss standardizing to one set across all comms.

Reply:
Let's add that in a second PR so we can get this one merged.